README.md

```diff
@@ -9,8 +9,7 @@ This is an elixir clone of the ruby gem [scientist](https://github.com/github/sc
 
 ## Wait, why be a Scientist?
 
-Suppose you decide to add a new caching layer to your production application, while still being able to make the same guarantees about your data. By processing your new
-caching strategy though Scientist you'll be able to
+Suppose you decide to add a new caching layer to your production application, while still being able to make the same guarantees about your data. By processing your new caching strategy through Scientist you'll be able to
```

Comment:
ughh, whitespace messed this diff 😞. 
This bit is about "caching strategy **though** Scientist" -> "caching strategy **through** Scientist"